src/CSET/operators/collapse.py

```diff
@@ -116,31 +118,43 @@ def collapse_by_lead_time(
     if method == "PERCENTILE":
         collapsed_cube = collapse(
             cube_to_collapse,
-            "forecast_period",
+            "forecast_reference_time",
             method,
             additional_percent=additional_percent,
         )
     else:
-        collapsed_cube = collapse(cube_to_collapse, "forecast_period", method)
+        collapsed_cube = collapse(cube_to_collapse, "forecast_reference_time", method)
     return collapsed_cube
 
 
 def collapse_by_hour_of_day(
-    cube: iris.cube.Cube,
+    cube: iris.cube.Cube | iris.cube.CubeList,
     method: str,
     additional_percent: float = None,
+    multi_case: bool = True,
     **kwargs,
 ) -> iris.cube.Cube:
     """Collapse a cube by hour of the day.
 
+    Collapses a cube by hour of the day in the time coordinates provided by the
+    model. It is useful for creating diurnal cycle plots. It aggregates all
+    00 UTC together regardless of lead time.
+
     Arguments
     ---------
-    cube: iris.cube.Cube
-        Cube to collapse and iterate over one dimension. It should contain only
-        one time dimension.
+    cube: iris.cube.Cube | iris.cube.CubeList
+        Cube to collapse and iterate over one dimension or CubeList to
+        convert to a cube and then collapse prior to aggregating by hour.
+        If a CubeList is provided multi_case must be True. A cube that only
+        contains one time dimension must have multi_case set to False; a cube
+        containing two time dimensions, e.g., 'forecast_reference_time' and
+        'forecast_period' must have multi_case set to True.
     method: str
         Type of collapse i.e. method: 'MEAN', 'MAX', 'MIN', 'MEDIAN',
         'PERCENTILE'. For 'PERCENTILE' the additional_percent must be specified.
+    multi_case: boolean
```

Comment:
```suggestion
    multi_case: boolean, optional
```
Currently this is an optional variable, defaulting to False. Do we want to keep it that way, or do we want it to be explicitly provided?

Reply:
Keep as optional, as that way saves specification effort on user, but is documented so they know what is happening (a sensible halfway house between fully specification by user and automation as I view it).